pom.xml

```diff
@@ -1718,9 +1718,8 @@
                 <plugin>
                     <groupId>org.apache.maven.plugins</groupId>
                     <artifactId>maven-surefire-plugin</artifactId>
-                    <version>3.0.0-M5</version>
+                    <version>3.0.0-M6</version>
                     <configuration>
-                        <forkNode implementation="org.apache.maven.plugin.surefire.extensions.SurefireForkNodeFactory"/>
```

Comment:
This workaround should be no longer needed with the newer surefire version: [SUREFIRE-1983](https://issues.apache.org/jira/browse/SUREFIRE-1983).